code/modules/mining/mine_items.dm

```diff
@@ -1353,3 +1353,49 @@ var/list/total_extraction_beacons = list()
 	for(var/turf/simulated/mineral/M in range(7,drill_loc))
 		if(prob(75))
 			M.GetDrilled(1)
+
+/****************Himeo Voidsuit Kit*****************/
+/obj/item/himeo_kit
+	name = "himeo voidsuit kit"
+	contained_sprite = TRUE
+	icon = 'icons/obj/mining_contained.dmi'
+	icon_state = "himeo_kit"
+	item_state = "himeo_kit"
+	desc = "A simple cardboard box containing the requisition forms, permits, and decal kits for a Himean voidsuit."
+	desc_fluff = "As part of a cost-cutting and productivity-enhancing initiative, NanoTrasen has authorized a number of Himean Type-76 'Fish Fur'\
+	for use by miners originating from the planet. Most of these suits are assembled in Cannington and painstakingly optimized on-site by their\
+	individual operator leading to a large trail of red tape as NanoTrasen is forced to inspect these suits to ensure their safety."
+	desc_info = "In order to convert a mining voidsuit into a Himean voidsuit, simply click on this box with a voidsuit or helmet in hand.\
+	The same process can be used to convert a Himean voidsuit back into a regular voidsuit. Make sure not to have a helmet or tank in the suit\
+	or else it will be deleted."
+	w_class = ITEMSIZE_SMALL
+
+/obj/item/himeo_kit/attackby(obj/item/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/clothing/suit/space/void/mining))
```

Comment:
these should eject any connect tanks, helmets, and magboots before deleting

Reply:
and also any attached accesories

Reply:
Heavy disagree. It should error if it attempts to repaint such a voidsuit. Allowing it to remove attachments and the like will result in it dismantling things which are not meant to be dismantled.